heartbeat_server/http_requests.py

```
@@ -168,6 +238,16 @@ def is_supported_http_request(data: bytearray, logger: Logger):
         logger.error(f'Exception is_supported_http_request: {exec}')
         return False
 
+def inspect_http_content_type(body):
+    try:
+        request = json.loads(body.decode('utf-8'))
+        if 'command' in request:
+            return 'remote'
+        if 'ccu' in request:
+            return 'add_meter'
+    except Exception as exec:
+        logger.error(f'Exception inspect_http_content_type: {exec}')
```

@thormiwa When would this exception be thrown? It may be better to raise this Exception if the two elements being checked here are not found in the payload.

---

@Reagan Another if block specifying if the two elements are not in the payload then to raise exception so then the try block can be removed?

---

OK @thormiwa . As long as the raised exception error is passed back in some form to the `AddMeter` request.